### DISCUSSION TITLE
Could not find requested route.

### DISCUSSION THREAD

**Discussion by speendo** [shlinkio/discussions#98] [Help wanted] [answered]
Hello and thank you for Shlink!

I just installed Shlink 3.0.3 using docker-compose and a PostgreSQL database.

I tried to create a short link using the CLI - it seems to work:

    $ docker exec -it shlink-app shlink short-url:create https://www.reddit.com/
    Processed long URL: https://www.reddit.com/
    Generated short URL: https://s.<mydoma.in>/MUuBZ

However, when I visit https://s.<mydoma.in>/MUuBZ, I only get this error screen
![image](https://user-images.githubusercontent.com/1668429/158799042-883988a4-f20f-475d-ad86-591b54ed79ce.png)
Can you guide me the way on how to find out what I did wrong?

Thank you!

**Comment by speendo** [comment, marked as answer]
Ok, seems that I could make it work.

I previously had the same subdomain pointing to the Shlink web interface and the Shlink app. This seems to cause some troubles.

Even when I changed the settings to something else (web app at short.doma.in and app at s.doma.in) the problematic settings seem to be cached and therefore still cause troubles. But on other computers it works already.

**Reply by tomshomelab** [reply]
could you explain a little more about implimenting the solution? how would i ajust my settings / where would i ajust my settings to fix this. im using docker-compose
